cancer_ai/validator/competition_handlers/melanoma_handler.py

```diff
@@ -4,26 +4,36 @@
 from typing import List
 from PIL import Image
 import numpy as np
-from sklearn.metrics import accuracy_score, precision_score, recall_score, confusion_matrix, roc_curve, auc
+from sklearn.metrics import (
+    accuracy_score,
+    precision_score,
+    recall_score,
+    confusion_matrix,
+    roc_curve,
+    auc,
+)
+
 
 class MelanomaCompetitionHandler(BaseCompetitionHandler):
-    """
-    """
+    """ """
```

Comment:
are we planning on to add documentation string here in future?

Reply:
in the future yes ^^